tests/framework/installer/ceph_installer.go

```diff
@@ -781,7 +781,7 @@ func (h *CephInstaller) removeClusterFinalizers(namespace, clusterName string) {
 	// Get the latest cluster instead of using the same instance in case it has been changed
 	cluster, err := h.k8shelper.RookClientset.CephV1().CephClusters(namespace).Get(ctx, clusterName, metav1.GetOptions{})
 	if err != nil {
-		logger.Errorf("failed to remove finalizer. failed to get cluster. %+v", err)
+		logger.Errorf("failed to get cluster. %v", err)
```

Comment:
Seems like you can revert this change, the original message is accurate, right? The finalizer failed to be removed.